ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt

```diff
@@ -23,7 +23,7 @@ import java.io.*
 import java.util.concurrent.*
 
 @State(Scope.Group)
-abstract class AsyncIntegrationBenchmark<TEngine : ApplicationEngine> {
+public abstract class AsyncIntegrationBenchmark<TEngine : ApplicationEngine> {
```

Comment:
Can't we simply add benchmarks module to the exclusion list? This module is not used outside and looks more like a testing module? Or if we want to keep our codebase uniform, shouldn't we avoid exclusions instead?

Reply:
Let's exclude benchmarks. It was accidentally touched by regex.

Reply:
Fixed in the fixup in the last PR